flit_core/flit_core/wheel.py

```diff
@@ -189,6 +189,14 @@ def copy_module(self):
             rel_path = osp.relpath(full_path, source_dir)
             self._add_file(full_path, rel_path)
 
+    def add_pth(self):
+        pth_target = osp.dirname(osp.abspath(self.module.path))
```

Comment:
We've got `self.module.source_dir` as a shortcut for this directory - you just need to `str()` it.